src/main.rs

```diff
@@ -81,8 +81,19 @@ fn main() -> Result<(), Box<dyn Error>> {
         );
     let matches = match opts.parse(std::env::args()) {
         Ok(it) => it,
+        Err(getopts::Fail::UnrecognizedOption(string)) => {
+            println!(
+                "{}",
+                opts.usage(&format!(
+                    "Unknown option '{string}'! Please rerun with some of these:"
+                    // "Unknown option '{string}'. The configured options are as follows:"
+                    // "Unresolvable argument: '{string}'. Valid options are as follows:"
+                ))
+            );
+            // Quit the application with `EX_USAGE` from `sysexits.h`.
+            std::process::exit(64);
```

Comment:
The last thing should be the exit with the `sysexits.h` code.  Shall I incorporate the corresponding crate therefore?

Reply:
Sorry, I got the message too late.  But I found some other places to apply the exit codes.  I am unsure whether you agree with my choices for the exit reasons since there are multiple options, sometimes.